ui-core/src/styles/inputs/inputWithSuggestions.css

```diff
@@ -0,0 +1,47 @@
+.input-with-suggestions {
+  position: relative;
+  width: 100%;
+  color: #312e2b;
+  font-size: 1rem;
+
+  .suggestions-list {
+    position: absolute;
+    width: calc(100% - 2.1rem);
+    top: calc(100% - 1.4rem);
+    left: 1rem;
+    right: 0;
+    z-index: 1000;
+    list-style: none;
+    max-height: calc(2.5rem * var(--number-of-suggestions));
+    overflow-y: auto;
+    background-color: #fff;
+    border-radius: 0.313rem;
+    border-radius: 6px 6px 6px 6px;
+    border-radius: 0px 0px 6px 6px;
+    border-radius: 6px;
+    box-shadow:
+      0px 6px 21px -5px rgba(24, 68, 239, 0.26),
+      0px 8px 30px -5px rgba(0, 0, 0, 0.16),
+      0px 3px 5px -2px rgba(0, 0, 0, 0.1);
+  }
+
+  .suggestion-item {
+    padding: 12px 19px;
```

Comment:
For better centering, in sketch the padding is 11px in top and 13px in bottom.

When`small` is true, these values are not the same, we should add another style for that.

Values should be in rem.